extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java

```diff
@@ -327,4 +330,16 @@ ServiceStartBuildItem startLiquibase(LiquibaseRecorder recorder,
         return Collections.emptySet();
     }
 
+    private Optional<String> extractChangeFile(Change change) {
+        if (change instanceof LoadDataChange) {
+            return Optional.of(((LoadDataChange) change).getFile());
+        }
+        if (change instanceof SQLFileChange) {
+            return Optional.of(((SQLFileChange) change).getPath());
+        }
+        if (change instanceof CreateProcedureChange) {
+            return Optional.of(((CreateProcedureChange) change).getPath());
+        }
+        return Optional.empty();
```

Comment:
AFAICS, `CreateViewChange` can also link to a file:
- https://www.liquibase.org/javadoc/liquibase/change/core/CreateViewChange.html#getPath--
- https://docs.liquibase.com/change-types/community/create-view.html

Reply:
See #14500